src/Option.purs

```diff
@@ -2549,6 +2549,12 @@ empty = Option Foreign.Object.empty
 -- | option4 :: Option.Option ( foo :: Boolean, bar :: Int )
 -- | option4 = Option.fromRecord { qux: [] }
 -- | ```
+-- |
+-- | You can also use `Option.fromRecord` to go from a `Record _` of `Maybe _` values to an `Option _`, e.g.:
```

Comment:
A minor stylistic change to look a little closer to the other stuff before this.

```suggestion
-- | You can also use `Option.fromRecord` to go from a `Record _` of `Maybe _` values to an `Option _`.
-- |
-- | E.g. The following definition is valid.
```